### ISSUE TITLE
Add bulk life-support recycler

### ISSUE THREAD

**Issue by jd284** [UmbraSpaceIndustries/MKS#1293]
The USI-LS system is designed to require one high-efficiency recycler (like the MKS Kerbitats) to set the efficiency cap, and a larger number of low efficiency recyclers like the RT-500 for supporting higher crew numbers.

So it would be nice if MKS had a high-throughput, low-efficiency bulk recycler with similar stats to the RT-500 (in terms of recycler mass per kerbal supported), because currently by far the most effective way of life-support recycling is spamming dozens of RT-500. The MKS recyclers (like the kerbitat and colonization modules) all focus on providing that high-effiency cap recycler, rather than throughput.

Basically, I'd suggest to create a 2.5m Tundra module equivalent to a dozen RT-500 or so, which given the RT-500 mass of 100 kg would still only be 1.2 t. To fit better into MKS, instead of base module mass this could also be using machinery instead (if the USI-LS recycling module supports this, that is). Then also scale this up to 3.75m for some serious recycling throughput as would be needed by large colonies and colony ships.

**Comment by UberWaffe** [comment]
I would suggest increasing the number of Kerbals supported by the RT-5000.
`RT-500 Recycling Module
Weight: 100kg / kerbal
EC: .5 / kerbal
Capital Cost: 100 / kerbal
Recycle percent: 60% (Extends 100 units of supplies to equivalent of 250)`

`RT-5000 Recycling Module
Weight: 1250kg / kerbal (x12.5 times more than RT-500)
EC: 6.25 / kerbal (x12.5 times more than RT-500)
Capital Cost: 1666.7 / kerbal (x16.6 times more than RT-500)
Recycle percent: 79% (Extends 100 units of supplies to equivalent of 476.2. About x1.9 better than RT-500)`

While the RT-5000 is a nearly double the efficiency of supplies over RT-500, it still falls far short in terms of weight and energy cost.

I personally adjust the RT-5000 to also recycle @ 60%, but then support up to 25 kerbals. (Still overall less efficient per Kerbal, but less hassle and part count, so meh.)
`Adjusted RT-5000 Recycling Module for 25 Kerbals
Weight: 150kg / kerbal (x1.5 times more than RT-500)
EC: .75 / kerbal (x1.5 times more than RT-500)
Capital Cost: 200 / kerbal (x2 times more than RT-500)
Recycle percent: 60% (Same as RT-500)`
